src/helpers/tests/measureTitleHelper.spec.ts

```diff
@@ -205,4 +205,98 @@ describe("measureTitleHelper", () => {
+
+            return matchingMeasure === undefined ? undefined : matchingMeasure.measure.title;
+        }
+
+        function getAliasOfMeasure(
+            visualizationObject: IVisualizationObjectContent,
+            localIdentifier: string,
+        ): string {
+            const measureBucketItems = visualizationObject.buckets[0].items;
+            const matchingMeasure: IMeasure = measureBucketItems
+                .map(bucketItem => bucketItem as IMeasure)
+                .find(bucketItem => bucketItem.measure.localIdentifier === localIdentifier);
+
+            return matchingMeasure === undefined ? undefined : matchingMeasure.measure.alias;
+        }
+
+        it("should delete all measures' titles", () => {
+            const visualizationObjectContent = findVisualizationObjectFixture("Arithmetic measures");
+            const result = ignoreTitles(visualizationObjectContent);
+
+            expect(
+                getTitleOfMeasure(result, "arithmetic_measure_created_from_complicated_arithmetic_measures"),
+            ).toBeUndefined();
+
+            expect(getTitleOfMeasure(result, "m1")).toBeUndefined();
+
+            expect(getTitleOfMeasure(result, "m2")).toBeUndefined();
+
+            expect(getTitleOfMeasure(result, "m3")).toBeUndefined();
+
+            expect(getTitleOfMeasure(result, "m4")).toBeUndefined();
+
+            expect(getTitleOfMeasure(result, "m1_pop")).toBeUndefined();
+
+            expect(getTitleOfMeasure(result, "m1_previous_period")).toBeUndefined();
+
+            expect(getTitleOfMeasure(result, "m1_pop_renamed")).toBeUndefined();
+
+            expect(getTitleOfMeasure(result, "m1_previous_period_renamed")).toBeUndefined();
+
+            expect(getTitleOfMeasure(result, "derived_measure_from_arithmetic_measure")).toBeUndefined();
+
+            expect(
+                getTitleOfMeasure(result, "arithmetic_measure_created_from_simple_measures"),
+            ).toBeUndefined();
+
+            expect(
+                getTitleOfMeasure(result, "arithmetic_measure_created_from_renamed_simple_measures"),
+            ).toBeUndefined();
+
+            expect(
+                getTitleOfMeasure(result, "arithmetic_measure_created_from_derived_measures"),
+            ).toBeUndefined();
+
+            expect(
+                getTitleOfMeasure(result, "arithmetic_measure_created_from_arithmetic_measures"),
+            ).toBeUndefined();
+
+            expect(
+                getTitleOfMeasure(result, "arithmetic_measure_created_from_renamed_derived_measures"),
+            ).toBeUndefined();
+
+            expect(
+                getTitleOfMeasure(result, "invalid_arithmetic_measure_with_missing_dependency"),
+            ).toBeUndefined();
+
+            expect(
+                getTitleOfMeasure(result, "invalid_arithmetic_measure_with_cyclic_dependency_1"),
+            ).toBeUndefined();
+
+            expect(
```

Comment:
Please use` it.each` for this case. You can refer to example code in other test files.

Reply:
Fixed!